mods/ts/sequences/misc.yaml

```diff
@@ -62,9 +62,12 @@ crate-effects:
 	levelup: veteran
 
 rank:
-	rank: pips
+	veteran: pips
 		Start: 7
-		Length: 2
+		Length: 1
```

Comment:
Isn't `Length: 1` already the default? https://github.com/OpenRA/OpenRA/blob/bea2b5cc91c3992f47aab955b94888b769855c9e/OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs#L180